parts/Compound/CompoundDialogs/index.js

```diff
@@ -177,7 +206,7 @@ const CompoundDialogs = ({
           {renderButton()}
         </Grid>
       </div>
-      <SnowStepBox transactionStatus={transactionStatus}/>
+      {<SnowStepBox transactionStatus={transactionStatus} title={title}/>}
```

Comment:
<SnowStepBox transactionStatus={transactionStatus} title={title}/>

Reply:
we can add that to a cleanup task later, the SnowStepBox DOM already has treatment for this